core/variant/variant_call.cpp

```diff
@@ -1928,18 +1930,28 @@ static void _register_variant_builtin_methods() {
 	bind_method(Basis, inverse, sarray(), varray());
 	bind_method(Basis, transposed, sarray(), varray());
 	bind_method(Basis, orthonormalized, sarray(), varray());
+	bind_method(Basis, orthogonalized, sarray(), varray());
```

Comment:
Does this method also make sense in `Transform2D` that also has a `orthonormalized` method?

Reply:
I think there are some cases that we want to keep the scale and only do orthogonalization, I'll take a look at it later.

Reply:
Added.